<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherDailyCard</class>
 <widget class="QWidget" name="WeatherDailyCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 主窗口背景 */
QWidget  {
    background-color: #e6f7ff;
    font-family: &quot;Microsoft YaHei&quot;;
    color: #2e3c4d;
    font-size: 14px;
}

QWidget#WeatherDailyCard{
    background-color: #e6f7ff;
    font-family: &quot;Microsoft YaHei&quot;;
    color: #2e3c4d;
    font-size: 14px;
    border-radius: 12px;
    border: 3px solid #000000;
}


/* 标签字体样式 */
QLabel {
    color: #2e3c4d;
    font-size: 14px;
}

/* 日期标签（可加类名或 objectName 细分） */
QLabel[date=&quot;true&quot;] {
    font-weight: bold;
    font-size: 16px;
}

/* 温度、风速、湿度说明 */
QLabel[data=&quot;true&quot;] {
    font-size: 13px;
    color: #444;
}

/* 天气图标可用 QLabel 设置 padding */
QLabel[icon=&quot;true&quot;] {
    padding: 10px;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_date">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_icons">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_temp">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_winds">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_humi">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
